lib/shanty_docker_plugin/image_wrapper.rb

```diff
@@ -0,0 +1,108 @@
+require 'docker'
+require 'shanty_docker_plugin/error'
+require 'shanty_docker_plugin/output_parser'
+require 'shanty/logger'
+
+module DockerPlugin
+  # Public: Provides wapper utility around Docker::Image
+  class ImageWrapper
+    include Shanty::Logger
```

Comment:
Making this a mixin is turning out to be a good decision for uses like this. :smile: